### PR TITLE
Fix bug 1141298 - Deprecate UserFindForm

### PR DESCRIPTION
**Pull request by blossomica**
I've fixed this bug. 
If anything is wrong you can let me know.

### CODE REVIEW

File: airmozilla/manage/views/users.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def users(request):
             User.objects.filter(_mozilla_email_filter).count()
         ),
     }
-    form = forms.UserFindForm()
+    form = forms.EventEditForm()
```
Comment (peterbe) [inline airmozilla/manage/views/users.py]: You might want to review that. It's not right.

**Comment by blossomica** [comment]
All right, can you explain bit more on what's the problem with it?

**Comment by blossomica** [comment]
Hi @peterbe , I tried to make some changes, I might be wrong again I am not sure. This is my first time working with django so I guess I need to read more on it to improve because I find it hard in some way. Any suggestions of things I can read will be appreciated.

File: airmozilla/manage/views/users.py [hunk 2]
```diff
@@ -39,12 +39,7 @@ def users(request):
             User.objects.filter(_mozilla_email_filter).count()
         ),
     }
-    form = forms.UserFindForm()
-    context = {
-        'form': form,
-        'users_stats': users_stats,
-    }
-    return render(request, 'manage/users.html', context)
```
Comment (peterbe) [inline airmozilla/manage/views/users.py]: Keep the `context` variable. But just remove the `'form': form,` line. (and the `form = forms.UserFindForm()` line of course)

**Comment by blossomica** [comment]
Oh okay, that's pretty much what I did in my last pull request [https://github.com/blossomica/airmozilla/commit/21b7743b2fdae4cdd2da5e63154acac8f914db49] but I didn't keep the context variable. So I will add the variable back.

**Comment by peterbe** [comment]
Wondeful! You're in!

**Comment by blossomica** [comment]
Awesome!

**Pull request merged**